Common/Data/UniverseSelection/UserDefinedUniverse.cs

```diff
@@ -65,13 +70,14 @@ public override UniverseSettings UniverseSettings
         /// <param name="securityInitializer">Initializes securities when they're added to the universe</param>
         /// <param name="interval">The interval at which selection should be performed</param>
         /// <param name="symbols">The initial set of symbols in this universe</param>
+        [Obsolete("This constructor is obsolete because SecurityInitializer is obsolete and will not be used.")]
         public UserDefinedUniverse(SubscriptionDataConfig configuration, UniverseSettings universeSettings, ISecurityInitializer securityInitializer, TimeSpan interval, IEnumerable<Symbol> symbols)
             : base(configuration, securityInitializer)
         {
             _interval = interval;
             _symbols = symbols.ToHashSet();
             _universeSettings = universeSettings;
-            _selector = time => _symbols;
+            _selector = time => _subscriptionDataConfigs.Select(x => x.Symbol).Union(_symbols);
```

Comment:
Was this `Union(_symbols)` required? Perhaps in the event a user implements a function that just returns symbols and we wouldn't have the requisite `SubscriptionDataConfig` instances? perhaps a comment here will suffice to make it clearer.

Or we can place the comment in the non-obsoleted constructor.

Reply:
I'll add a new comment explaining it 👍